pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -127,16 +127,52 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
+message PodSecurityContext {
+    // The configurations for the sandbox's namespaces.
+    // This will be used only if the PodSandbox uses namespace for isolation.
+    optional NamespaceOption namespace_options = 2;
+    // Optional SELinux context to be applied.
+    optional SELinuxOption selinux_options = 3;
+    // The UID to run the entrypoint of the container process.
+    // Defaults to user specified in image metadata if unspecified.
+    // May also be set in PodSecurityContext.  If set in both SecurityContext and
+    // PodSecurityContext, the value specified here takes precedence.
+    optional int64 run_as_user = 4;
+    // Indicates that the container must run as a non-root user.
+    // If true, the runtime should validate the image at runtime to ensure that it
+    // does not run as UID 0 (root) and fail to start the container if it does.
+    // If unset or false, no such validation will be performed.
+    // May also be set in PodSecurityContext.  If set in both SecurityContext and
+    // PodSecurityContext, the value specified here takes precedence.
+    optional bool run_as_non_root = 5;
+    // If set, the root filesystem of the container is read-only.
+    optional bool readonly_rootfs = 6;
+    // A list of groups applied to the first process run in each container, in addition
+    // to the container's primary GID.  If unspecified, no groups will be added to
+    // any container.
+    repeated int64 supplement_groups = 7;
```

Comment:
Rechecked with this, I think `supplemental_groups` is better (keep same with api)